lib/functions/transfer.js

```diff
@@ -70,7 +70,7 @@ class Transfer extends AsyncGeneratorFunction {
                     throw Error(`'contentRange.length' too large, not supported yet: ${transferPart.source.url}`);
                 }
 
-                console.log(`Transferred content range: ${contentRange}`);
+                console.log(`Transferred content range: ${JSON.stringify(contentRange)}`);
```

Comment:
are you certain this will always be an object?

Reply:
Even if it's not an object - in the case of a string it's a NO-OP